README.md

```diff
@@ -32,6 +32,16 @@ _.append(5, [1, 2, 3, 4]);
 // => [1, 2, 3, 4, 5]
 ```
 
+### assign :: [Object] -> Object
```

Comment:
Yeah, I wasn't sure about that but I definitely prefer `mergeAll` as a name. Do we need `merge` though if you can just pass in an array of two objects to get the same functionality?